bot/pagination.py

```diff
@@ -51,22 +51,25 @@ def __init__(
         suffix: str = '```',
         max_size: int = 2000,
         scale_to_size: int = 2000,
-        max_lines: t.Optional[int] = None
+        max_lines: t.Optional[int] = None,
+        linesep: str = "\n"
     ) -> None:
         """
         This function overrides the Paginator.__init__ from inside discord.ext.commands.
 
         It overrides in order to allow us to configure the maximum number of lines per page.
         """
-        self.prefix = prefix
-        self.suffix = suffix
-
         # Embeds that exceed 2048 characters will result in an HTTPException
         # (Discord API limit), so we've set a limit of 2000
         if max_size > 2000:
             raise ValueError(f"max_size must be <= 2,000 characters. ({max_size} > 2000)")
 
-        self.max_size = max_size - len(suffix)
+        super().__init__(
+            prefix,
+            suffix,
+            max_size - len(suffix),
+            linesep
+        )
```

Comment:
I'm not sure why calling the init function of the parent class seems better than doing it manually. Additionally this _should_ have been a separate PR.

Reply:
This is a breaking change in d.py 1.7 so needs to be part of this PR.

Calling the parent constructor was just for simplicity.

Reply:
Sure, but wouldn't it be cleaner to call the functions manually? Since that's the way it was done before

Reply:
This same change was made when sir-lancebot was upgraded to 1.17

Reply:
Sure, but this isn't a PR that bumps the version

Reply:
Anyhow, this _is_ kinda nitpicky 😅

Reply:
We definitely don't need a separate PR for every small change, the feature needed a version bump of a dependency so that was done.
Calling the super class' init feels better to me as in case the underlying init changes we won't be storing the attributes in a possibly incompatible way